src/mcts/search.cc

```diff
@@ -150,17 +153,24 @@ void Search::SendUciInfo() REQUIRES(nodes_mutex_) {
       cache_->GetSize() * 1000LL / std::max(cache_->GetCapacity(), 1);
   uci_info_.nps =
       uci_info_.time ? (total_playouts_ * 1000 / uci_info_.time) : 0;
-  uci_info_.score = 290.680623072 * tan(1.548090806 * best_move_edge_.GetQ(0));
-  uci_info_.pv.clear();
+  uci_info_.comment.clear();
 
-  bool flip = played_history_.IsBlackToMove();
-  for (auto iter = best_move_edge_; iter;
-       iter = GetBestChildNoTemperature(iter.node()), flip = !flip) {
-    uci_info_.pv.push_back(iter.GetMove(flip));
-    if (!iter.node()) break;  // Last edge was dangling, cannot continue.
+  int pv_idx = 0;
+  for (auto edge_ptr = asc_move_edges_.rbegin(); 
+         edge_ptr != asc_move_edges_.rend(); ++edge_ptr) {
```

Comment:
Couldn't this loop just be written as `for (const auto& edge : asc_move_edges) {}` ?

Reply:
Would have written it like if I didn't but one of the lists was naturally ascending so had to use reverse iterators.